modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -461,6 +461,73 @@ definition {
 		ObjectPortlet.viewEntry(entry = "Entry B");
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Add a custom object, field, and add a relationship itself") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject",
+				objectName_2 = "CustomObject",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+		}
+
+		task ("Define custom field as title field on the object") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+			CreateObject.selectTitleField(fieldLabel = "Custom Field");
+
+			CreateObject.saveObject();
+		}
+
+		task ("Add 3 entries by API to relate to other entries") {
+			for (var entryName : list "A,B,C") {
+				ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+					fieldName = "customObjectField",
+					objectName = "CustomObject",
+					value = "Entry ${entryName}");
+			}
+		}
+
+		task ("Add one-to-many relationships with entries and add additional relationships") {
+			Navigator.openURL();
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+			for (var entryName : list "B,C") {
+				ObjectPortlet.viewEntryDetails(entry = "Entry ${entryName}");
+
+				ObjectPortlet.selectRelationshipFieldEntry(
+					entry = "",
+					newEntry = "Entry A");
+
+				CreateObject.assertEntrySelectedOnRelationshipField(entry = "Entry A");
+
+				Navigator.gotoBack();
+
+				ObjectPortlet.viewEntry(entry = "Entry A");
+			}
+		}
```

Comment:
Since we're testing the relationship can be edited, we should be changing the relationship here. Something like this:
1. Create four entries
2. Relate Entry A to Entry B and Entry C
4. On the table, assert Entry A is listed as the relationship for Entry B and Entry C
5. Relate Entry D to Entry B and Entry C
7. On the table, assert Entry D is listed as the relationship for Entry B and Entry C
8. On the table, assert Entry A is not listed as the relationship for Entry B and Entry C

We would need a new path to find a specific relationship on the table, either based on position or the `following-sibling` which contains the entry name. I can explain more tomorrow during our sync.

Reply:
@evanilson-santana:

Reply:
Use a time box and don't spend any more than half a day on this. Ideally, we go with the third solution but the other two are ok as well.